src/elements/content-sidebar/activity-feed/task-new/__tests__/__snapshots__/AssigneeDetails-test.js.snap

```diff
@@ -0,0 +1,178 @@
+<Component
+  completedAt="2019-01-01T00:00:000Z"
+  status="APPROVED"
+  user={
+    Object {
+      "id": "123",
+      "name": "user one",
+    }
+  }
+>
+  <div
+    className="bcs-AssigneeDetails"
+  >
+    <div>
+      user one
+    </div>
+    <div
+      className="bcs-AssigneeDetails-status"
+    >
+      <FormattedMessage
+        defaultMessage="Approved {dateTime}"
+        id="be.contentSidebar.activityFeed.taskNew.tasksFeedStatusApproved"
+        values={
+          Object {
+            "dateTime": <Datestamp
+              date="2019-01-01T00:00:000Z"
+            />,
+          }
+        }
+      >
+        <div />
+      </FormattedMessage>
+    </div>
+  </div>
+</Component>
+`;
+
+exports[`elements/content-sidebar/ActivityFeed/task-new/AssigneeDetails should render details for status APPROVED 2`] = `
+<Component
+  completedAt={2019-01-01T00:00:00.000Z}
+  status="APPROVED"
+  user={
+    Object {
+      "id": "123",
+      "name": "user one",
+    }
+  }
+>
+  <div
+    className="bcs-AssigneeDetails"
+  >
+    <div>
+      user one
+    </div>
+    <div
+      className="bcs-AssigneeDetails-status"
+    >
+      <FormattedMessage
+        defaultMessage="Approved {dateTime}"
+        id="be.contentSidebar.activityFeed.taskNew.tasksFeedStatusApproved"
+        values={
+          Object {
+            "dateTime": <Datestamp
+              date={2019-01-01T00:00:00.000Z}
+            />,
+          }
+        }
+      >
+        <div />
+      </FormattedMessage>
+    </div>
+  </div>
+</Component>
+`;
+
+exports[`elements/content-sidebar/ActivityFeed/task-new/AssigneeDetails should render details for status NOT_STARTED 1`] = `
+<Component
+  completedAt="2019-01-01T00:00:000Z"
+  status="NOT_STARTED"
+  user={
+    Object {
+      "id": "123",
+      "name": "user one",
+    }
+  }
+>
+  <div
+    className="bcs-AssigneeDetails"
+  >
+    <div>
+      user one
+    </div>
+  </div>
+</Component>
+`;
+
+exports[`elements/content-sidebar/ActivityFeed/task-new/AssigneeDetails should render details for status REJECTED 1`] = `
+<Component
+  completedAt="2019-01-01T00:00:000Z"
+  status="REJECTED"
+  user={
+    Object {
+      "id": "123",
+      "name": "user one",
+    }
+  }
+>
+  <div
+    className="bcs-AssigneeDetails"
+  >
+    <div>
+      user one
+    </div>
+    <div
+      className="bcs-AssigneeDetails-status"
+    >
+      <FormattedMessage
+        defaultMessage="Rejected {dateTime}"
+        id="be.contentSidebar.activityFeed.taskNew.tasksFeedStatusRejected"
+        values={
+          Object {
+            "dateTime": <Datestamp
+              date="2019-01-01T00:00:000Z"
+            />,
+          }
+        }
+      >
+        <div />
+      </FormattedMessage>
+    </div>
+  </div>
+</Component>
+`;
+
+exports[`elements/content-sidebar/ActivityFeed/task-new/AssigneeDetails should render details for status invalid_string 1`] = `
+<Component
+  completedAt="2019-01-01T00:00:000Z"
+  status="invalid_string"
```

Comment:
Is this a valuable snapshot? Should it not render the component if the status is invalid or something other behavior?